### DISCUSSION TITLE
建议增加社区功能，用户自己可以在社区发布组件。这个组件是通过element来组成的或者构成的。

### DISCUSSION THREAD

**Discussion by 233cy** [element-plus/element-plus#16558] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

建议增加社区功能，用户自己可以在社区发布组件。这个组件是通过element来组成的或者构成的。
可能说是一个插件库更合适。
这样用户可以发布一些在elm封装好的一些组件库，如：快速选择另一个数据表中的某条数据、下拉选择可快速新增插件等。

### Additional comments

_(empty)_

**Comment by polarove** [comment]
这会带来额外的维护成本

**Comment by xxholly32** [comment]
自己封装就好了 比如：https://github.com/tolking/element-pro-components

**Comment by jinceon** [comment]
和npm有什么区别？
官方也没办法审核大家的组件质量。就是单纯比Npm范围小了点而已
